enrich/geo/index.js

```diff
@@ -0,0 +1,123 @@
+const dot = require("dotenv")
+const path = require("path")
+const isEmpty = require("lodash/isEmpty")
+const Airtable = require("airtable")
+
+dot.config({
+  path: path.resolve(path.join(__dirname, "/../../.env.development")),
+})
+
+const { retrieveGeoForOrg } = require("./retrieveGeo")
+const { computeGeo } = require("./computeGeo")
+
+async function asyncForEach(array, callback) {
+  const promises = []
+  for (let index = 0; index < array.length; index += 1) {
+    promises.push(callback(array[index], index, array))
+  }
+  return Promise.all(promises)
+}
+
+function getFieldsGeoData(fields) {
+  return [
+    fields.Country,
+    fields.State,
+    fields.City,
+    fields.Longitude,
+    fields.Latitude,
+  ]
+}
+
+async function main() {
+  if (!process.env.AIRTABLE_API_KEY) {
+    console.log("Missing AIRTABLE_API_KEY env property")
+    return
+  }
+
+  const base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base(
+    process.env.AIRTABLE_BASE_ID
+  )
+
+  // const table = base("Organizations")
+  const table = base("Organizations")
+  table
+    .select({
+      pageSize: 10,
```

Comment:
Curious why you went with a small page size here?

Reply:
the update method only allow batch of 10 maximum, maybe I should have comment it :)

Reply:
@loiclefloch got it, this makes sense. I took a different approach in mine that we might be able to incorporate here. I query all the organizations at once then buffer any updates into an array that gets flushed any time it fills up (10 items) or at the end if any are left. The benefit is that we always make full use of our update queries, reducing the total number of updates we send and speeding up the script.